environments/pull-environments.sh

```diff
@@ -21,5 +21,5 @@ if [ "$NODE_ENV" = "test" ]; then
   numParallelTasks=1; # GitHub actions has trouble handling parallel executions
 fi
 
-echo $dirs | xargs -P $numParallelTasks -I {} sh -c "./pull-environment.sh {}"
+echo $dirs | xargs -P $numParallelTasks -I {} sh -c "../pull-environment.sh {}"
```

Comment:
I understand this is relative to the cwd of each environment folder, but it is a little confusing that we're referencing a file in the same directory as this script using `..`. I wonder if it's easy to reference `pull-environment.sh` from the path of `pull-environments.sh`.

Reply:
That's a good point; we can get the directory that `pull-environments.sh` is in with:

```bash
shell_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
```

and then we can do:

```suggestion
echo $dirs | xargs -P $numParallelTasks -I {} sh -c ""$shell_path"/pull-environment.sh {}"
```

Reply:
(source: https://stackoverflow.com/a/24112741)

Reply:
actually that shell command can prob be simplified... taking a look.

Reply:
I this that approach is a bit cleaner and allows us to run the `pull-environments.sh` script from any directory, but I'd say timebox it as I think it's just a nice-to-have :)

Reply:
Agreed, done! [d506c78](https://github.com/aws-amplify/amplify-ui/pull/2502/commits/d506c7871cc97c404366ff67ccdab135918d9224)